R/fa_i.R

```diff
@@ -24,19 +41,54 @@
 #' @export
 fa_i <- function(name,
                  class = NULL,
+                 htmlDependency = NULL,
```

Comment:
I think that if we keep this argument, it should go after the `...`.

Reply:
Now done.